lib/logstash/plugin_mixins/kafka/avro_schema_registry.rb

```diff
@@ -24,6 +24,24 @@ def setup_schema_registry_config
       # This option permits to define a proxy to be used to reach the schema registry service instance.
       config :schema_registry_proxy, :validate => :uri
 
+      # If schema registry client authentication is required, this setting stores the keystore path.
+      config :schema_registry_ssl_keystore_location, :validate => :string
```

Comment:
```suggestion
      config :schema_registry_ssl_keystore_path, :validate => :string
```

Do you think we should stick to the current approach, making config names as close as possible to Kafka's properties, or should we (kind of) apply the new plugin's naming convention for SSL? if we decide to keep it close to Kafka, IMO, we should also exclude this plugin from the upcoming standardization plans.

Reply:
This naming was chosen to reflect the existing `ssl_keystore_location` but I agree, I'll adopt the new TLS naming convention

Reply:
Suggestion applied

Reply:
Actually, my comment's main intention was to discuss the SSL standardization for this plugin and if we should follow or not the new SSL naming convention here.

Considering this plugin's settings are already aligned with Kafka properties names, and that those settings are very particular and well-known by Kafka's users, I would keep your previous version, matching Kafka's names.
We've discussed the SSL standardization offline (@jsvd @yaauie @roaksoax)  and agreed on skipping (at least for now) Kafka plugins due to that reason.

If you agree, I think we could roll back your last commit and keep it as it's.

Reply:
I think that adding new settings is better to avoid debt, and stick immediately with new naming conventions.
So if we all agree on this, we can keep this naming for new setting. A follow up PR will be created deprecate the existing ones and introduce the new naming here.

Reply:
Did we get an agreement on this? @jsvd?

Reply:
Ah, I missed this one, thanks @edmocosta.
@andsel we decided we should leave the Kafka integration outside of the TLS normalization process since there are naming conventions users of Kafka will be familiar with.
So I'd prefer us to use `_location` instead of `_path`, sorry for the back and forth.

Reply:
Reverted with 5dc98c5b586eea826336ac58ae9d8c9ad3dced25